isis/appdata/import/MerMI.tpl

```diff
@@ -0,0 +1,201 @@
+    InstrumentID                = {{ INSTRUMENT_ID.Value }}
+    InstrumentName              = "{{ INSTRUMENT_NAME.Value }}"
+    InstrumentSerialNumber      = {% if exists("INSTRUMENT_SERIAL_NUMBER") %}
+                                  {{ INSTRUMENT_SERIAL_NUMBER.Value }}
+                                  {% else %}
+                                  NULL
+                                  {% endif %}
+    LocalTrueSolarTime          = {% if exists("LOCAL_TRUE_SOLAR_TIME") %}
+                                  {{ LOCAL_TRUE_SOLAR_TIME.Value }}
+                                  {% else %}
+                                  NULL
+                                  {% endif %}
+    PlanetDayNumber             = {% if exists("PLANET_DAY_NUMBER") %}
+                                  {{ PLANET_DAY_NUMBER.Value }}
+                                  {% else %}
+                                  NULL
+                                  {% endif %}
+    SolarLongitude              = {% if exists("SOLAR_LONGITUDE") %}
+                                  {{ SOLAR_LONGITUDE.Value }}
+                                  {% else %}
+                                  NULL
+                                  {% endif %}
+    SpacecraftClockCntPartition = {% if exists("SPACECRAFT_CLOCK_CNT_PARTITION") %}
+                                  {{ SPACECRAFT_CLOCK_CNT_PARTITION.Value }}
+                                  {% else %}
+                                  NULL
+                                  {% endif %}
+    SpacecraftClockStartCount   = {% if exists("SPACECRAFT_CLOCK_START_COUNT") %}
+                                  {{ SPACECRAFT_CLOCK_START_COUNT.Value }}
+                                  {% else %}
+                                  NULL
+                                  {% endif %}
+    SpacecraftClockStopCount    = {% if exists("SPACECRAFT_CLOCK_STOP_COUNT") %}
+                                  {{ SPACECRAFT_CLOCK_STOP_COUNT.Value }}
+                                  {% else %}
+                                  NULL
+                                  {% endif %}
+    StartTime                   = {% if exists("START_TIME") %}
+                                  {% set startTime=START_TIME.Value %}
+                                  {{ RemoveStartTimeZ(startTime) }}
+                                  {% else %}
+                                  NULL
+                                  {% endif %}
+    StopTime                    = {% if exists("STOP_TIME") %}
+                                  {% set stopTime=STOP_TIME.Value %}
+                                  {{ RemoveStartTimeZ(stopTime) }}
+                                  {% else %}
+                                  NULL
+                                  {% endif %}
+    ExposureDuration            = {% if exists("INSTRUMENT_STATE_PARMS.EXPOSURE_DURATION") %}
+                                  {{ INSTRUMENT_STATE_PARMS.EXPOSURE_DURATION.Value }}<ms>
```

Comment:
Should there be a space between `{{ INSTRUMENT_STATE_PARMS.EXPOSURE_DURATION.Value }}` and `<ms>`?

Reply:
Good catch. It seems like the label ingestion is still handling this correctly. But for consistency, I can add in a space.